### PR TITLE
Add QCASH logo create 0xe74b35425fe7e33ea190b149805baf31139a8290

### PR DESCRIPTION
**Pull request by ZzikoWang**
This PR is being closed due to inactivity. If you wish to continue, please reopen the PR before sending your payment.
Do NOT send payments for closed PR, as the fee may by lost!
  (70 vs. 60 hrs)

### CODE REVIEW

File: blockchains/ethereum/assets/0xe74b35425fe7e33ea190b149805baf31139a8290 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ZzikoWang) [inline blockchains/ethereum/assets/0xe74b35425fe7e33ea190b149805baf31139a8290:1]: logo
add QCASH logo

**Comment by Jeffcardi** [comment]
> add QCASH logo.png
0xb8Ee0723fe3e5860D63da7C4c9095933AA92664c

**Review by Jeffcardi — APPROVED** [review]
0xb8Ee0723fe3e5860D63da7C4c9095933AA92664c

**Comment by catenocrypt** [comment]
Could you please add the logo as file `blockchains/ethereum/assets/0xe74b35425fe7e33ea190b149805baf31139a8290/logo.png` ?

Also, please add an `info.json` file next to the `logo.png`, with info about the project, of the form:

```json
{
    "name": "Trust Wallet Token",
    "website": "https://trustwallet.com",
    "short_description": "Utility token to increase adoption of cryptocurrency.",
    "explorer": "https://explorer.binance.org/asset/TWT-8C2"
}
```

**Comment by ZzikoWang** [comment]
Sorry, I can't upload it?
github error
GitHub error2

**Comment by ZzikoWang** [comment]
> 您能否将徽标添加为文件`blockchains/ethereum/assets/0xe74b35425fe7e33ea190b149805baf31139a8290/logo.png`？
> 
> 另外，请在`info.json`旁边添加一个文件`logo.png`，其中包含有关该项目的信息，格式为：
> 
> ```json
> {
>      “名称”：“信任钱包令牌”，
>      “网站”：“ https://trustwallet.com ”，
>      “ short_description ”：“实用令牌以增加对加密货币的采用。”，
>      “浏览器”：“ https：// explorer .binance.org / asset / TWT-8C2 “ 
> }
> ```
Hello, it has been added. Please help to have a look

**Comment by ZzikoWang** [comment]
> 您能否将徽标添加为文件`blockchains/ethereum/assets/0xe74b35425fe7e33ea190b149805baf31139a8290/logo.png`？
> 
> 另外，请在`info.json`旁边添加一个文件`logo.png`，其中包含有关该项目的信息，格式为：
> 
> ```json
> {
>      “名称”：“信任钱包令牌”，
>      “网站”：“ https://trustwallet.com ”，
>      “ short_description ”：“实用令牌以增加对加密货币的采用。”，
>      “浏览器”：“ https：// explorer .binance.org / asset / TWT-8C2 “ 
> }
> ```

Hello, I've paid. Can you take a look at it for me?
bnb-github

**Comment by ZzikoWang** [comment]
https://explorer.binance.org/tx/4387D73B2362C3A916F74825D9FCB0C3170850546DD3FF9170F535F70EF25302

**Review by ZzikoWang — COMMENTED** [review]
Delete empty files

**Comment by ZzikoWang** [comment]
image
https://explorer.binance.org/address/bnb1tqq9llyr3dyjd559dha6z5r5etk3qfwk07m098
TxHash：4387D73B2362C3A916F74825D9FCB0C3170850546DD3FF9170F535F70EF25302
Our payment should be normal. I want to know why merge-fee-bot does not recognize.

**Comment by catenocrypt** [comment]
Reopen

**Comment by catenocrypt** [comment]
Accepting payment manually 
https://explorer.binance.org/tx/4387D73B2362C3A916F74825D9FCB0C3170850546DD3FF9170F535F70EF25302
(payment was not picked up by bot due to review; now it is too old)

**Pull request merged**